PWGCF/TableProducer/filterCorrelations.cxx

```diff
@@ -255,6 +269,19 @@ struct FilterCF {
     auto bc = collision.template bc_as<aod::BCsWithTimestamps>();
     outputCollisions(bc.runNumber(), collision.posZ(), collision.multiplicity(), bc.timestamp());
 
+    if constexpr (std::experimental::is_detected<HasMultTables, typename T1::iterator>::value) {
+      multiplicities.clear();
+      if (cfgEstimatorBitMask & kCentFT0C)
+        multiplicities.push_back(collision.centFT0C());
+      if (cfgEstimatorBitMask & kMultFV0A)
+        multiplicities.push_back(collision.multFV0A());
+      if (cfgEstimatorBitMask & kMultNTracksPV)
+        multiplicities.push_back(collision.multNTracksPV());
+      if (cfgEstimatorBitMask & kMultNTracksGlobal)
+        multiplicities.push_back(collision.multNTracksGlobal());
+      outputMultSets(multiplicities);
+    }
+
```

Comment:
It does not matter much but in principle this code could be in `processDataMults`. Nothing should speak against filling one table there and the then one would not need the `HasMultTables`, or am missing something?